pangeo_forge/utils.py

```diff
@@ -9,3 +12,17 @@ def chunked_iterable(iterable, size):
         if not chunk:
             break
         yield chunk
+
+
+@task
```

Comment:
This being a task makes it play with with `mapped` inputs, see https://github.com/pangeo-forge/noaa-oisst-avhrr-feedstock/pull/3/files#diff-c1ba9cb202886813ca3e7aa5645ec022ebf0588625b8411e5ed004e81e1b707aR117-R121.

I suspect that using `mapped` inputs will be pretty common.